src/query/test/block.go

```diff
@@ -124,16 +124,23 @@ func NewBlockFromValuesWithSeriesMeta(
 }
 
 // NewBlockFromValuesWithMetaAndSeriesMeta creates a new block using the provided values
+// Note: panics on errors, since this is a testutil
```

Comment:
Would be nice if this took in a `t.Testing` and failed that but totally not worth the effort :p

Reply:
Haha yeah that was my thought as well.